.github/workflows/helpers.js

```diff
@@ -90,6 +90,22 @@ const isWithinRCP = ({ offset = 0, excludeShortRCP = false } = {}) => {
   return false;
 };
 
+const isWithinPrePostRCP = ({ offset = 7 } = {}) => {
```

Comment:
I think this could be simplified. Say we have an RCP that lasts between the 14th and the 21st. I'm not sure we need to concern ourselves with what happens between those two dates, we're just interested in having a larger batch size between the 7th and the 28th. This would eliminate the first conditional and trim down the final return statement.